*** tests/A-01-11-M-ok.out	2021-11-06 09:35:15.599020000 +0000
--- tests/A-01-11-M-ok.outhyp	2021-11-19 15:02:43.098297054 +0000
***************
*** 45,52 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! NEW|ROLHA|10
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 45,51 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Mostrar parceiro: Operação inválida: O parceiro 'M2' não existe.
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
***************
*** 55,62 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
! NEW|ROLHA|10
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 54,60 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Mostrar parceiro: Operação inválida: O parceiro 'S1' não existe.
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
