*** tests/A-03-08-M-ok.out	2021-10-24 17:09:38.535816000 +0100
--- tests/A-03-08-M-ok.outhyp	2021-11-18 18:17:25.216978263 +0000
***************
*** 50,56 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|M1|SAL|10|10|10|2
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 50,56 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|M1|SAL|10|0|0|ggc.core.Date@1fb3ebeb|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
