*** tests/A-04-08-M-ok.out	2021-10-24 17:46:04.420193000 +0100
--- tests/A-04-08-M-ok.outhyp	2021-11-19 20:10:42.819471615 +0000
***************
*** 28,33 ****
--- 28,34 ----
  0 - Sair
  Escolha uma opção: HIDROGENIO|S1|2|5000
  ROLHA|M1|2|500
+ SAL|S1|1|10
  VIDRO|M1|1|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
***************
*** 44,49 ****
--- 45,51 ----
  4 - Visualizar lotes de um dado produto
  0 - Sair
  Escolha uma opção: Identificador do parceiro: HIDROGENIO|S1|2|5000
+ SAL|S1|1|10
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
