*** tests/A-10-10-M-ok.out	2021-11-17 12:10:08.582594676 +0000
--- tests/A-10-10-M-ok.outhyp	2021-11-17 17:38:15.112167198 +0000
***************
*** 42,48 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|500|0|0
  M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
--- 42,48 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|0|0|0
  M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
***************
*** 54,60 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
- BARGAIN|SAL|50
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 54,59 ----
***************
*** 64,70 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
- BARGAIN|SAL|50
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 63,68 ----
