*** tests/A-10-20-M-ok.out	2021-11-17 12:10:08.582594676 +0000
--- tests/A-10-20-M-ok.outhyp	2021-11-17 18:18:51.112590821 +0000
***************
*** 53,59 ****
  0 - Sair
  Escolha uma opção: HIDROGENIO|200|5000
  ROLHA|2000|500
! SAL|10|10
  VIDRO|1000|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
--- 53,60 ----
  0 - Sair
  Escolha uma opção: HIDROGENIO|200|5000
  ROLHA|2000|500
! SAL|10|0
! SAL|10|0
  VIDRO|1000|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
***************
*** 139,145 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
  3 - Registar venda
--- 140,147 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Data limite de pagamento: Identificador do produto: Quantidade: Registar venda: Operação inválida: Produto 'SAL': pedido=10, existências=0
! Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
  3 - Registar venda
***************
*** 180,189 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
- NEW|SAL|10
- BARGAIN|ROLHA|100
- NEW|SAL|20
- BARGAIN|ROLHA|50
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 182,187 ----
***************
*** 193,202 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
- NEW|SAL|10
- BARGAIN|ROLHA|100
- NEW|SAL|20
- BARGAIN|ROLHA|50
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 191,196 ----
***************
*** 206,213 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M3|M3|M3 add|NORMAL|0|0|0|0
- NEW|SAL|20
- BARGAIN|ROLHA|50
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 200,205 ----
***************
*** 217,224 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M4|M4|M4 end|NORMAL|0|0|0|0
- NEW|SAL|20
- BARGAIN|ROLHA|50
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 209,214 ----
