*** tests/A-10-20-M-ok.out	2021-11-05 13:01:36.152472000 +0000
--- tests/A-10-20-M-ok.outhyp	2021-11-18 18:17:30.644863849 +0000
***************
*** 53,58 ****
--- 53,60 ----
  0 - Sair
  Escolha uma opção: HIDROGENIO|200|5000
  ROLHA|2000|500
+ SAL|10|0
+ SAL|10|0
  SAL|10|10
  VIDRO|1000|500
  Gestão de Produtos
***************
*** 139,145 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Data limite de pagamento: Identificador do produto: Quantidade: Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
  3 - Registar venda
--- 141,148 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Data limite de pagamento: Identificador do produto: Quantidade: Registar venda: Operação inválida: Produto 'SAL': pedido=10, existências=0
! Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
  3 - Registar venda
***************
*** 180,189 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
- NEW|SAL|10
- BARGAIN|ROLHA|100
- NEW|SAL|20
- BARGAIN|ROLHA|50
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 183,188 ----
***************
*** 193,202 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
- NEW|SAL|10
- BARGAIN|ROLHA|100
- NEW|SAL|20
- BARGAIN|ROLHA|50
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 192,197 ----
***************
*** 206,213 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M3|M3|M3 add|NORMAL|0|0|0|0
- NEW|SAL|20
- BARGAIN|ROLHA|50
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 201,206 ----
***************
*** 217,224 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M4|M4|M4 end|NORMAL|0|0|0|0
- NEW|SAL|20
- BARGAIN|ROLHA|50
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 210,215 ----
