*** tests/A-10-20-M-ok.out	2021-11-05 13:01:36.152472000 +0000
--- tests/A-10-20-M-ok.outhyp	2021-11-18 20:59:56.623122452 +0000
***************
*** 54,59 ****
--- 54,60 ----
  Escolha uma opção: HIDROGENIO|200|5000
  ROLHA|2000|500
  SAL|10|10
+ SAL|10|10
  VIDRO|1000|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
***************
*** 180,189 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: S1|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
- NEW|SAL|10
- BARGAIN|ROLHA|100
- NEW|SAL|20
- BARGAIN|ROLHA|50
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 181,186 ----
***************
*** 193,202 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
- NEW|SAL|10
- BARGAIN|ROLHA|100
- NEW|SAL|20
- BARGAIN|ROLHA|50
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 190,195 ----
***************
*** 206,213 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M3|M3|M3 add|NORMAL|0|0|0|0
- NEW|SAL|20
- BARGAIN|ROLHA|50
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 199,204 ----
***************
*** 217,224 ****
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
  Escolha uma opção: Identificador do parceiro: M4|M4|M4 end|NORMAL|0|0|0|0
- NEW|SAL|20
- BARGAIN|ROLHA|50
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 208,213 ----
