*** tests/A-11-11-M-ok.out	2021-10-26 19:36:30.487098000 +0100
--- tests/A-11-11-M-ok.outhyp	2021-11-19 14:58:00.487388464 +0000
***************
*** 105,111 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|80|2100|VIDRO:1#ROLHA:1
  ROLHA|100|620
  VIDRO|100|560
  Gestão de Produtos
--- 105,111 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|80|2000|VIDRO:1#ROLHA:1
  ROLHA|100|620
  VIDRO|100|560
  Gestão de Produtos
***************
*** 114,122 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|M1|40|50
! GARRAFA|S1|20|50
! GARRAFA|S1|50|1000
  GARRAFA|S1|80|1000
  ROLHA|M1|10|100
  ROLHA|M1|20|500
--- 114,120 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|S1|50|1000
  GARRAFA|S1|80|1000
  ROLHA|M1|10|100
  ROLHA|M1|20|500
