*** tests/A-11-04-M-ok.out	2021-10-21 19:36:19.803330000 +0100
--- tests/A-11-04-M-ok.outhyp	2021-11-18 20:40:01.867680250 +0000
***************
*** 42,48 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: COMPRA|0|M1|GARRAFA|1000|70000|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 42,48 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: COMPRA|0|M1|GARRAFA|1000|70|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
***************
*** 96,103 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|M1|70|1000
! GARRAFA|S1|50|1000
  GARRAFA|S1|80|1000
  ROLHA|M1|20|500
  VIDRO|M1|30|500
--- 96,102 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|S1|50|1000
  GARRAFA|S1|80|1000
  ROLHA|M1|20|500
  VIDRO|M1|30|500
