*** tests/A-11-04-M-ok.out	2021-10-21 19:36:19.803330000 +0100
--- tests/A-11-04-M-ok.outhyp	2021-11-18 21:43:40.201326070 +0000
***************
*** 42,48 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: COMPRA|0|M1|GARRAFA|1000|70000|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 42,48 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: COMPRA|0|M1|GARRAFA|1000|70|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
