*** tests/A-11-10-M-ok.out	2021-10-26 19:26:21.121282000 +0100
--- tests/A-11-10-M-ok.outhyp	2021-11-19 13:59:03.153527911 +0000
***************
*** 40,46 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|80|2000|VIDRO:1#ROLHA:1
  ROLHA|10|0
  VIDRO|30|500
  Gestão de Produtos
--- 40,46 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|80|2000|nullVIDRO:1#ROLHA:1
  ROLHA|10|0
  VIDRO|30|500
  Gestão de Produtos
***************
*** 99,105 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|80|2000|VIDRO:1#ROLHA:1
  ROLHA|10|10
  VIDRO|30|500
  Gestão de Produtos
--- 99,105 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|80|2000|nullVIDRO:1#ROLHA:1
  ROLHA|10|10
  VIDRO|30|500
  Gestão de Produtos
