*** tests/A-11-11-M-ok.out	2021-10-26 19:36:30.487098000 +0100
--- tests/A-11-11-M-ok.outhyp	2021-11-18 19:37:13.481002471 +0000
***************
*** 105,113 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|80|2100|VIDRO:1#ROLHA:1
! ROLHA|100|620
! VIDRO|100|560
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
--- 105,119 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|80|8000|nullVIDRO:1#ROLHA:1
! GARRAFA|80|8050
! GARRAFA|40|50
! ROLHA|20|2000
! ROLHA|20|2100
! ROLHA|100|20
! VIDRO|30|2000
! VIDRO|100|2010
! VIDRO|40|50
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
