*** tests/A-11-08-M-ok.out	2021-10-26 19:16:21.143782000 +0100
--- tests/A-11-08-M-ok.outhyp	2021-11-18 19:37:12.980996750 +0000
***************
*** 23,29 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: COMPRA|0|M1|ROLHA|100|10000|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 23,29 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: COMPRA|0|M1|ROLHA|100|100|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 48,55 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|80|2000|VIDRO:1#ROLHA:1
! ROLHA|100|2100
  VIDRO|90|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
--- 48,56 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|80|2000|nullVIDRO:1#ROLHA:1
! ROLHA|50|4000
! ROLHA|100|100
  VIDRO|90|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
***************
*** 110,116 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: COMPRA|1|M1|GARRAFA|100|30000|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 111,117 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: COMPRA|1|M1|GARRAFA|100|300|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 173,180 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|300|2100|VIDRO:1#ROLHA:1
! ROLHA|100|2100
  VIDRO|90|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
--- 174,183 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|80|4000|nullVIDRO:1#ROLHA:1
! GARRAFA|300|100
! ROLHA|50|4000
! ROLHA|100|100
  VIDRO|90|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
