*** tests/A-11-05-M-ok.out	2021-10-21 22:33:00.626383000 +0100
--- tests/A-11-05-M-ok.outhyp	2021-11-19 13:59:02.377538568 +0000
***************
*** 48,54 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|80|2000|VIDRO:1#ROLHA:1
  PAPEL|12|2000
  ROLHA|20|500
  VIDRO|30|500
--- 48,54 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|80|2000|nullVIDRO:1#ROLHA:1
  PAPEL|12|2000
  ROLHA|20|500
  VIDRO|30|500
