*** tests/A-11-11-M-ok.out	2021-10-26 19:36:30.487098000 +0100
--- tests/A-11-11-M-ok.outhyp	2021-11-18 19:56:52.479893489 +0000
***************
*** 105,129 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|80|2100|VIDRO:1#ROLHA:1
! ROLHA|100|620
! VIDRO|100|560
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|M1|40|50
! GARRAFA|S1|20|50
! GARRAFA|S1|50|1000
  GARRAFA|S1|80|1000
- ROLHA|M1|10|100
  ROLHA|M1|20|500
- ROLHA|S1|100|20
  VIDRO|M1|30|500
- VIDRO|M1|40|50
- VIDRO|M2|100|10
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
--- 105,123 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|80|8000|nullVIDRO:1#ROLHA:1
! ROLHA|100|2000
! VIDRO|100|2000
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|S1|50|1000
  GARRAFA|S1|80|1000
  ROLHA|M1|20|500
  VIDRO|M1|30|500
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
***************
*** 131,138 ****
  4 - Visualizar lotes de um dado produto
  0 - Sair
  Escolha uma opção: Identificador do produto: VIDRO|M1|30|500
- VIDRO|M1|40|50
- VIDRO|M2|100|10
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
--- 125,130 ----
