*** tests/A-12-10-M-ok.out	2021-11-12 10:32:47.530213000 +0000
--- tests/A-12-10-M-ok.outhyp	2021-11-19 20:28:27.304498710 +0000
***************
*** 83,89 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: -50
  Saldo contabilístico: 0
  Menu Principal
  1 - Abrir
--- 83,89 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: -470
  Saldo contabilístico: 0
  Menu Principal
  1 - Abrir
***************
*** 121,127 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: -150
  Saldo contabilístico: -100
  Menu Principal
  1 - Abrir
--- 121,127 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: -570
  Saldo contabilístico: -100
  Menu Principal
  1 - Abrir
