*** tests/A-12-07-M-ok.out	2021-11-17 12:10:08.582594676 +0000
--- tests/A-12-07-M-ok.outhyp	2021-11-17 17:38:20.288514529 +0000
***************
*** 80,87 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 20
! Saldo contabilístico: 238
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 80,87 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 0
! Saldo contabilístico: 0
  Menu Principal
  1 - Abrir
  2 - Guardar
***************
*** 118,125 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 53
! Saldo contabilístico: 238
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 118,125 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 0
! Saldo contabilístico: 0
  Menu Principal
  1 - Abrir
  2 - Guardar
