*** tests/A-12-06-M-ok.out	2021-11-17 12:10:08.582594676 +0000
--- tests/A-12-06-M-ok.outhyp	2021-11-18 17:12:22.235843511 +0000
***************
*** 69,76 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 130
! Saldo contabilístico: 200
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 69,76 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 0
! Saldo contabilístico: 0
  Menu Principal
  1 - Abrir
  2 - Guardar
***************
*** 107,114 ****
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 180
! Saldo contabilístico: 200
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 107,114 ----
  8 - Menu de Consultas
  9 - Ver Saldo Global
  0 - Sair
! Escolha uma opção: Saldo disponível: 0
! Saldo contabilístico: 0
  Menu Principal
  1 - Abrir
  2 - Guardar
***************
*** 127,133 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|1|M2|GARRAFA|10|50|50|2|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 127,133 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|1|M2|GARRAFA|10|0|0|ggc.core.Date@31befd9f|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 135,141 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|M1|ROLHA|10|20|20|1
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 135,141 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|M1|ROLHA|10|0|0|ggc.core.Date@1c20c684|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
