*** tests/A-12-05-M-ok.out	Fri Nov 12 10:11:14 2021
--- tests/A-12-05-M-ok.outhyp	Wed Nov 17 12:12:05 2021
***************
*** 56,62 ****
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 0
! Saldo contabilístico: 200
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 56,62 ----
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 0
! Saldo contabilístico: 0
  Menu Principal
  1 - Abrir
  2 - Guardar
