*** tests/A-12-05-M-ok.out	2021-11-12 10:11:14.110332000 +0000
--- tests/A-12-05-M-ok.outhyp	2021-11-19 18:50:45.669002935 +0000
***************
*** 56,62 ****
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 0
! Saldo contabilístico: 200
  Menu Principal
  1 - Abrir
  2 - Guardar
--- 56,62 ----
  9 - Ver Saldo Global
  0 - Sair
  Escolha uma opção: Saldo disponível: 0
! Saldo contabilístico: 15
  Menu Principal
  1 - Abrir
  2 - Guardar
