*** tests/A-13-05-M-ok.out	2021-11-17 12:10:08.586594659 +0000
--- tests/A-13-05-M-ok.outhyp	2021-11-17 12:10:57.809502277 +0000
***************
*** 13,20 ****
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Preço: ROLHA|M1|30|500
! ROLHA|R1|20|500
  Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
--- 13,20 ----
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Preço: ROLHA|R1|20|500
! ROLHA|M1|30|500
  Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
