*** tests/A-14-05-M-ok.out	2021-11-01 23:11:20.620093000 +0000
--- tests/A-14-05-M-ok.outhyp	2021-11-19 18:38:29.326389830 +0000
***************
*** 109,116 ****
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: VENDA|1|M1|ROLHA|3|6|6|3|2
! VENDA|4|M1|SAL|5|30|30|3|2
  Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
--- 109,119 ----
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: COMPRA|3|M1|SAL|4|20|2
! VENDA|0|M1|SAL|2|20|0|3|0
! VENDA|1|M1|ROLHA|3|6|2|3|0
! VENDA|2|M1|HIDROGENIO|4|8|0|3|0
! VENDA|4|M1|SAL|5|25|5|3|0
  Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
