*** tests/A-14-02-M-ok.out	2021-10-28 17:47:43.270897000 +0100
--- tests/A-14-02-M-ok.outhyp	2021-11-18 20:36:42.513652602 +0000
***************
*** 48,54 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: COMPRA|0|M1|SAL|10|100|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 48,54 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: COMPRA|0|M1|SAL|10|10|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 71,77 ****
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
--- 71,79 ----
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: COMPRA|0|M1|SAL|10|10|0
! COMPRA|1|M1|VIDRO|20|10|0
! Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
