*** tests/A-14-03-M-ok.out	2021-10-28 17:53:32.408431000 +0100
--- tests/A-14-03-M-ok.outhyp	2021-11-19 13:59:05.781491012 +0000
***************
*** 45,51 ****
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
--- 45,53 ----
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: VENDA|0|M1|ROLHA|10|0|0|ggc.core.Date@77459877|0
! VENDA|1|M1|VIDRO|25|0|0|ggc.core.Date@5b2133b1|0
! Consultas
  1 - Produtos com Preço Abaixo de Limite
  2 - Facturas pagas por parceiro
  0 - Sair
