*** tests/A-15-06-M-ok.out	2021-10-31 22:20:20.042622000 +0000
--- tests/A-15-06-M-ok.outhyp	2021-11-19 13:59:07.185470864 +0000
***************
*** 16,22 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador do parceiro: Identificador do produto: Quantidade: Registar desagregação: Operação inválida: Produto 'GARRAFA': pedido=6, existências=5
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 16,31 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Gestão de Transacções
! 1 - Visualizar
! 2 - Registar desagregação
! 3 - Registar venda
! 4 - Registar compra
! 5 - Receber pagamento de venda
! 0 - Sair
! Escolha uma opção: Número inválido!
! Escolha uma opção: Número inválido!
! Escolha uma opção: Opção inválida!
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 41,47 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|200|5|ROLHA:1#SAL:5#VIDRO:10
  HIDROGENIO|200|5000
  ROLHA|2|500
  SAL|16|500
--- 50,56 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|200|5|nullROLHA:1#SAL:5#VIDRO:10
  HIDROGENIO|200|5000
  ROLHA|2|500
  SAL|16|500
