*** tests/A-17-02-M-ok.out	2021-11-04 19:04:13.916400000 +0000
--- tests/A-17-02-M-ok.outhyp	2021-11-18 21:00:02.834983141 +0000
***************
*** 50,56 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: COMPRA|0|M1|SAL|23|529|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 50,56 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: COMPRA|0|M1|SAL|23|23|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
