*** tests/A-20-05-M-ok.out	2021-11-17 12:10:08.586594659 +0000
--- tests/A-20-05-M-ok.outhyp	2021-11-18 17:12:29.351730367 +0000
***************
*** 44,50 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|M1|SAL|3|300|300|4|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 44,50 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|M1|SAL|3|0|0|ggc.core.Date@33c7353a|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 52,58 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|1|M2|GARRAFA|24|240|240|2|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 52,58 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|1|M2|GARRAFA|24|0|0|ggc.core.Date@681a9515|0
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 79,85 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: M1|Rohit Figueiredo|New Delhi, India|SELECTION|3000|0|300|300
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 79,85 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: M1|Rohit Figueiredo|New Delhi, India|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
***************
*** 88,94 ****
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|SELECTION|2400|0|240|240
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
--- 88,94 ----
  5 - Mostrar compras com parceiro
  6 - Mostrar vendas (e desagregações) com parceiro
  0 - Sair
! Escolha uma opção: Identificador do parceiro: M2|Toshiba|Tokyo, Japan|NORMAL|0|0|0|0
  Gestão de Parceiros
  1 - Mostrar parceiro
  2 - Mostrar parceiros
